radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -34,6 +33,7 @@ elseif(PCB STREQUAL X9D+)
   set(HAPTIC YES)
   set(LUA_EXPORT lua_export_x9d)
   set(FLAVOUR x9d+)
+  set (PWR_BUTTON_DEFAULT "SWITCH")
```

Comment:
It's not possible to have PRESS here

Reply:
That only sets the default the real setting is further down